Src/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -268,19 +281,21 @@ public void DownloadAsync_Cancel()
             }
         }
 
-        /// <summary>Tests that download reports errors.</summary>
+        /// <summary>Tests that download reports errors, deserializing a JSON response correctly.</summary>
         [Test]
-        public void Download_Error()
+        public void Download_Error_JsonResponse()
         {
-            var downloadUri = "http://www.sample.com?alt=media";

```

Comment:
Yes - because the downloader adds it itself. It turns out the test was *really* throwing an exception not due to the status code we were saying to return, but because the URI assertion was failing.